plugin/trino-hive/src/test/java/io/trino/plugin/hive/coercions/TestTimestampCoercer.java

```diff
@@ -86,11 +88,26 @@ public void testTimestampToSmallerVarchar()
         assertLongTimestampToVarcharCoercions(TIMESTAMP_PICOS, longTimestamp, createVarcharType(29), "2023-04-11 05:16:12.345678876");
     }
 
+    @Test
+    public void testHistoricalLongTimestampToVarchar()
+    {
+        LocalDateTime localDateTime = LocalDateTime.parse("1899-12-31T23:59:59.999999999");
+        SqlTimestamp timestamp = SqlTimestamp.fromSeconds(TIMESTAMP_PICOS.getPrecision(), localDateTime.toEpochSecond(UTC), localDateTime.get(NANO_OF_SECOND));
+        assertThatThrownBy(() -> assertLongTimestampToVarcharCoercions(
+                TIMESTAMP_PICOS,
+                new LongTimestamp(timestamp.getEpochMicros(), timestamp.getPicosOfMicros()),
+                createUnboundedVarcharType(),
+                "1899-12-31 23:59:59.999999999"))
+                .isInstanceOf(TrinoException.class)
+                .hasMessageContaining("Coercion on historical dates is not supported");
+    }
+
     @DataProvider
     public Object[][] timestampValuesProvider()
     {
         return new Object[][] {
                 // before epoch
+                {"1900-01-01T00:00:00.000", "1900-01-01 00:00:00"},
```

Comment:
add `0001-01-01 00:00` test case. this is favorite users' date

Reply:
For historical dates we don't support coercion so this is the closest favorite data